src/subapps/dataExplorer/DataExplorerTable.tsx

```diff
@@ -115,7 +115,11 @@ const defaultColumnConfig = (colName: string): ColumnType<Resource> => {
     title: getColumnTitle(colName),
     dataIndex: colName,
     className: `data-explorer-column data-explorer-column-${colName}`,
-    sorter: false,
+    sorter: (a, b) => {
+      return JSON.stringify(a[colName] ?? '').localeCompare(
+        JSON.stringify(b[colName] ?? '')
+      );
+    },
```

Comment:
I don't know it's just a thought, 
I think comparing projects will be more useful only to test the project part not the org part
so instead comparing `org/project` will only compare `project`